projects/1.18/assets/blockcarpentry/blockcarpentry/lang/zh_cn.json

```
@@ -28,10 +32,74 @@
   "block.blockcarpentry.frame_fence": "框架栅栏",
   "block.blockcarpentry.frame_trapdoor": "框架活板门",
   "block.blockcarpentry.frame_door": "框架门",
-  "block.blockcarpentry.frame_pressure_plate": "木制框架压板",
+  "block.blockcarpentry.frame_pressure_plate": "木制框架压力板",
   "block.blockcarpentry.frame_stairs": "框架楼梯",
   "block.blockcarpentry.frame_button": "木制框架按钮",
   "block.blockcarpentry.falling_frameblock": "框架（受重力影响-WIP）",
-  "block.blockcarpentry.frame_slab": "框架板",
-  "block.blockcarpentry.frameblock": "框架"
+  "block.blockcarpentry.frame_slab": "框架台阶",
+  "block.blockcarpentry.frameblock": "框架",
+  "block.blockcarpentry.frame_carpet": "框架地毯",
+  "block.blockcarpentry.frame_pane": "框架板",
+  "block.blockcarpentry.frame_daylight_detector": "框架阳光探测器",
+  "block.blockcarpentry.frame_layered_block": "框架分层方块",
+  "message.blockcarpentry.light_level": "光照等级：%s",
+  "message.blockcarpentry.texture": "纹理：%s",
+  "message.blockcarpentry.design": "设计：%s",
+  "message.blockcarpentry.design_texture": "设计纹理：%s",
+  "message.blockcarpentry.grass_overlay": "激活草覆盖层",
+  "message.blockcarpentry.grass_overlay_large": "激活高草丛覆盖层",
+  "message.blockcarpentry.snow_overlay_small": "激活薄雪覆盖层",
+  "message.blockcarpentry.snow_overlay": "激活雪覆盖层",
+  "message.blockcarpentry.vine_overlay": "激活藤蔓覆盖层",
+  "message.blockcarpentry.special_overlay": "激活特殊覆盖层%s",
+  "message.blockcarpentry.rotation": "旋转纹理%s",
+  "message.blockcarpentry.lock": "此方块当前已被锁定！",
+  "message.blockcarpentry.unlock": "此方块当前已解锁，玩家可以打开或关闭它！",
+  "message.blockcarpentry.redstone_off": "此方块当前忽略红石信号",
+  "message.blockcarpentry.redstone_on": "此方块不再忽略红石信号",
+  "message.blockcarpentry.block_not_available": "很抱歉，此方块无法显示",
+  "message.blockcarpentry.block_error": "此方块发生错误，请向模组作者（PianoManu）报告",
+  "message.blockcarpentry.converting_outdated_block": "此方块有一个过时的方块实体，我已经为你修复了！:)",
+  "advancements.blockcarpentry.root.description": "用你选择的方块右键点击它，然后玩得开心",
+  "advancements.blockcarpentry.illusion_blocks.title": "区别在哪？？？？？？",
```

| **审查级别** |
|--------------|
| ⚪ 轻微 / 不阻止合并但需三思 |

参考此模组的译法，https://www.mcmod.cn/class/3084.html

原文：**WhAt'S tHe dIfFeReNcE??????**
```suggestion
  "advancements.blockcarpentry.illusion_blocks.title": "区 别 在 哪？？？？？？",
```